lib/matplotlib/rcsetup.py

```diff
@@ -209,6 +209,20 @@ def validator(s):
     validate_float, doc='return a list of floats')
 
 
+def validate_marker(s):
```

Comment:
This doesn't need to be public.
```suggestion
def _validate_marker(s):
```